extmod/vfs_fat.c

```diff
@@ -317,6 +318,39 @@ STATIC mp_obj_t vfs_fat_umount(mp_obj_t self_in) {
 }
 STATIC MP_DEFINE_CONST_FUN_OBJ_1(fat_vfs_umount_obj, vfs_fat_umount);
 
+#if _USE_LABEL
```

Comment:
should this use MICROPY_FATFS_USE_LABEL instead?

Reply:
Sure!